tools/flaky-test-reporter/slack_notification.go

```diff
@@ -137,6 +136,19 @@ func createSlackMessageForRepo(rd *RepoData, flakyIssuesMap map[string][]*flakyI
 	return message
 }
 
+// getSlackChannel gets the channel(s) to send messages to for a given job configuration
+func getSlackChannel(repo string, name string) []slackChannel {
+	jobMap, ok := slackChannelsMap[repo]
+	if !ok {
+		log.Printf("cannot find Slack channel mapping for repo '%s', skipping Slack notification", repo)
```

Comment:
still need to return here, otherwise it will be a runtime error on next line
I will lgtm if this is fixed

Reply:
my bad, should be good to go now.